### ISSUE TITLE
luakit often fails to open links in the primary instance

### ISSUE THREAD

**Issue by DavPalma** [luakit/luakit#871]
## I'm submitting a…
Bug Report

## Current Behavior
Luakit frequently, though not always, fails to open a link in a new tab when a remote instance is started with unique.

## Expected Behavior
Luakit opens a new tab with with each url I request with luakit [url].

## Reproduction Instructions
1. open luakit requiring unique
2. type 'luakit www.gnome.org' on a terminal and enter
3, repeat 2, a few times
4. check how many tabs it opens
## Environment
Linux Distribution & Version: Parabola GNU/Linux
Output of `luakit --version`:
luakit 2.2.1-1-g916c31ea
  built with webkit 2.28.4 (installed version: 2.30.1)

**Comment by c0dev0id** [comment]
Are you getting errors/warnings on stdout?
Does it also happen if you wait between the luakit invocations?

I'm not really able to reproduce it here. I put 40 times "luakit www.gnome.org" in a for loop and it opened all 40 tabs (it takes a while though).

I remember that I had tabs missing when I did load tests with many many tabs, but then I got a warnings about them. I think this might have been a timing issue when too much webkit process forking is going on.

Maybe try the single process mode (enter `export WEBKIT_USE_SINGLE_WEB_PROCESS="TRUE"` before starting luakit the first time). Does this work better?
